build.gradle

```diff
@@ -3,12 +3,12 @@
 buildscript {
 
     ext {
-        release_version = '1.9.0'
-        release_code = 11
+        release_version = '1.10.0'
```

Comment:
Let's make it 2.0 since it's a breaking change

Reply:
I was also thinking about it :)